### DISCUSSION TITLE
ColourPicker default value doesn't work

### DISCUSSION THREAD

**Discussion by jsuckart** [ant-design/ant-design#46442] [Q&A]
Hey can anyone help me on this?

I am using the color picker to pick a color for a text, but it only works if I manually choose a colour.
But sometimes I might just want to leave the default colour, but then the color value submitted by the form will be undefined.
Am I missing something here?

Thanks

**Comment by yoyo837** [comment]
Are there any reproduction links? Please.

**Comment by b5414** [comment]
Use `<Form initialValues` instead of `<ColorPicker defaultValue` if you using form
